ext/posix/posix.stub.php

```diff
@@ -191,6 +191,34 @@
  */
 const POSIX_RLIMIT_INFINITY = UNKNOWN;
 #endif
+#ifdef _SC_ARG_MAX
+/**
+ * @var int
+ * @cvalue _SC_ARG_MAX
+ */
+const _SC_ARG_MAX = UNKNOWN;
```

Comment:
We should prefix posix extension constants with `POSTFIX` so maybe just having `POSIX_ARG_MAX` would be better.

Reply:
Or if you prefer to differentiate the SC ones, then maybe `POSIX_SC_ARG_MAX`